webpack.config.js

```diff
@@ -39,7 +39,7 @@ module.exports = (vars) => {
           type: 'javascript/auto',
         },
         {
-          test: /\.(png|svg|jpg|jpeg|gif|webp|avif|svg|woff|woff2)$/i,
+          test: /\.(png|js|svg|jpg|jpeg|gif|webp|avif|svg|woff|woff2)$/i,
```

Comment:
Why are we adding .js ( javascript ) files in here?

Reply:
a quick solution or as I SAY being a cowboy in order to keep code clean of fixutrees and other copy stuff